tsconfig.json

```
@@ -2,7 +2,7 @@
 	"extends": "./tsconfig.build.json",
 	"exclude": ["node_modules/**"],
 	"compilerOptions": {
-		"types": ["jest", "node"],
+		"types": ["node" /* , "jest" ] // uncomment this if using jest */],
```

```suggestion
		"types": ["node"]
```
this is using vitest instead of jest, so this can be removed entirely